File: JavaScript/1037. Valid Boomerang.js
/**
 * @param {number[][]} points
 * @return {boolean}
 */
var isBoomerang = function(points) {
    x1 = points[0][0];
    y1 = points[0][1];
    x2 = points[1][0];
    y2 = points[1][1];
    x3 = points[2][0];
    y3 = points[2][1];
    
    return (x3 - x2) * (y2 - y1) != (x2 - x1) * (y3 - y2)
};

File: Python/1037. Valid Boomerang.py
class Solution:
    def isBoomerang(self, points: List[List[int]]) -> bool:
        (x1, y1), (x2, y2), (x3, y3) = points
        # (x2 - x1) / (y2 - y1) == (x3 - x2) / (y3 - y2)
        return (x3 - x2) * (y2 - y1) != (x2 - x1) * (y3 - y2)
